content-services/content-sources-backend
# Fixes 1447: add postgres tasking system

I'll update this with a bunch of information and a todo.
## Summary

## Testing steps
1. Still working on a few things
2. If you add `new_tasking_system: True` to your config.yaml, it will use this for introspect instead of kafka.
3. Should work right now for all endpoints that trigger introspect (`create`, `update`, introspect`).
4. For now, if you want to test the tasking system more thoroughly I suggest modifying the introspect handler to do other things, then trigger it via the introspect endpoint.
